core/src/main/resources/hudson/PluginWrapper/PluginWrapperAdministrativeMonitor/message.jelly

```diff
@@ -4,25 +4,25 @@
     <form method="post" action="${rootURL}/${it.url}/act" name="${it.id}">
       <f:submit name="correct" value="${%Correct}"/>
     </form>
-    <p><strong>${%Dependency errors}:</strong></p>
     <p>${%blurbOriginal}</p>
+    <p><strong>${%DirectlyDependentPlugins.title}</strong></p>
     <j:forEach items="${it.plugins}" var="plugin">
       <j:if test="${plugin.hasOriginalDependencyErrors()}">
         <dl>
-          <dt>${plugin.longName} version ${plugin.version}</dt>
+          <dt>${plugin.longName} <span class="plain">(${plugin.version})</span></dt>
```

Comment:
since this is a new class anyways. Can you change the class name from "plain" (describes how it looks) to "plugin_version" (describes what it is)

Cause in the future if we want to make plugin versions bold and red, it'll be very tempting to make "plain" bold and red, then be inaccurate.

Reply:
```suggestion
          <dt>${plugin.longName} <span class="plugin_version">(${plugin.version})</span></dt>
```